projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -345,3 +345,12 @@ export function loginCartUser() {
   login(cartUser.registrationData.email, cartUser.registrationData.password);
   cy.url().should('not.contain', 'login');
 }
+
+export function waitForCartRefresh() {
```

Comment:
remove this and reuse the `registerCartRefreshRoute` that Marcin refactored (it's the same function) 

just change the function name to `registerCartRefreshRoute` in line 182 because somehow it got reverted to the old name